.mergify.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+# Format Ref: https://doc.mergify.io/configuration.html
+pull_request_rules:
+    - name: 'After Cirrus-CI Fails'
```

Comment:
Just a reference label, I don't think the name actually shows up anywhere.  Experimentation will be needed to know for sure.